### PR TITLE
Release 2.1.3

### PR DESCRIPTION
**Pull request by tduffield**
### Description
Update CHANGELOG and prep for 2.1.3 release

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [X] CHANGELOG.md has been updated if required (not required for bugfixes, required for API changes)
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ StackOverflow discussion that's relevant]
 
 - [ ] New functionality includes tests
 - [ ] All tests pass
-- [ ] RELEASE\_NOTES.md has been updated if required (not required for bugfixes, required for API changes)
+- [ ] CHANGELOG.md has been updated if required (not required for bugfixes, required for API changes)
```
Comment (btm) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you remove what is in the parenthesis here? If we just have `CHANGELOG.md`, we still want a line in there for a bug fix.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,155 +0,0 @@
-# Contributing to Chef
```
Comment (btm) [inline CONTRIBUTING.md]: I see that this file is replaced by a line in the `README.md`. That's probably fine, but Github treats the `CONTRIBUTING.md` as special and links to it when someone opens a PR, so I would keep it with the same line it in that is in the `README.md` pointing to chef/chef.

Comment (tduffield) [inline CONTRIBUTING.md]: That isn't what we've been doing across our other repositories. If we want to do that we should probably keep it consistent across the other projects as well (such as chef-dk, omnibus, ohai and many others).

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (tduffield) [inline .github/PULL_REQUEST_TEMPLATE.md]: Sure. Easy enough.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tduffield) [inline CONTRIBUTING.md]: @tas50 I know you were involved with this effort before. Any thoughts ^^ ?

Comment (btm) [inline CONTRIBUTING.md]: Yeah, no big deal, I wouldn't block too long on figuring it out. I'm happy to defer to @tas50, but the Github feature is nice.

https://github.com/blog/1184-contributing-guidelines

**Comment by tduffield** [comment]
DO NOT MERGE: There is a bug with Omnibus where builds are not being produced.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tas50) [inline CONTRIBUTING.md]: Just nuke the file and toss a contributing section with the link in the readme. Keeps it simple

**Comment by tduffield** [comment]
Okay. We should be clear to merge now. Was able to produce a build locally.

**Pull request merged**